python/ml4ir/build/Dockerfile

```diff
@@ -14,11 +14,10 @@ RUN pip config set global.extra-index-url https://pypi.python.org/simple
 RUN pip install --upgrade setuptools
 RUN pip install -r requirements.txt
 
-# Copy ml4ir
-COPY . /home/ml4ir/python/
+# Copy docker entrypoint
+COPY ml4ir/build/run_driver.sh $PYTHONPATH/.
 
 # Set docker entrypoint
-RUN cp $PYTHONPATH/ml4ir/build/run_driver.sh $PYTHONPATH/.
```

Comment:
`RUN cp` looks strange here. `COPY`?

Reply:
No, `cp` is right because it is an internal copy. `COPY` is to move files from your machine to the image.

Reply:
(I think that's what was being done)

Reply:
I just removed this guys 😀
I now only `COPY` (from project directory to Docker image) the `run_driver.sh` directly to where we want to find it.

The full ml4ir copy is not needed as I mount the volume now